crates/turbopack-ecmascript/src/references/esm/export.rs

```diff
@@ -340,6 +346,25 @@ pub struct EsmExports {
     pub star_exports: Vec<Vc<Box<dyn ModuleReference>>>,
 }
 
+#[turbo_tasks::value_impl]
+impl EsmExports {
+    #[turbo_tasks::function]
+    pub async fn expand_star_exports(&self) -> Result<Vc<Vec<Vc<ExpandResult>>>> {
+        let mut results = vec![];
+
+        for esm_ref in self.star_exports.iter() {
+            if let ReferencedAsset::Some(asset) =
+                &*ReferencedAsset::from_resolve_result(esm_ref.resolve_reference()).await?
```

Comment:
ReferencedAsset should only be used in code generation.

Reply:
What's the alternative to use here?